jobs/webcompat-kb/webcompat_kb/main.py

```diff
@@ -1289,7 +1295,11 @@ def run(self) -> None:
 
         kb_ids = list(kb_data.keys())
 
-        history_changes = self.fetch_update_history(all_bugs)
+        if self.include_history:
+            history_changes = self.fetch_update_history(all_bugs)
```

Comment:
nit: maybe `fetch_update_history` can be renamed to just `fetch_history` since the updating part was moved out?